examples/medplum-scheduling-demo/src/components/AppointmentActions.tsx

```diff
@@ -44,7 +44,17 @@ export function AppointmentActions(props: AppointmentActionsProps): JSX.Element
       await medplum.updateResource({
         ...appointment,
         status: newStatus,
```

Comment:
Use `PATCH` for these small, targetted changes

Reply:
Alright, I'll do this refactoring in another PR when I move the code to a bot.